gulp/tasks/content.js

```diff
@@ -0,0 +1,46 @@
+var path = require('path');
+var gulp = require('gulp');
+var clone = require('gulp-clone');
+var buffer = require('vinyl-buffer');
+var mergeStream = require('merge-stream');
+var generateContentAndTemplates = require('../util/content').generateContentAndTemplates;
+var through = require('through2');
+
+gulp.task('content', function() {
+  var base, contentJSON, languagesJSON, matchHTML, templates, templateStreams;
+
+  base = path.resolve(__dirname, '..', '..');
+  contentJSON = require('../../source/content.js');
+  languagesJSON = require('../../source/languages.js');
+  matchHTML = /\.html$/;
+  templates = {};
+  templateStreams = mergeStream();
```

Comment:
Did we want to block this on stylistic changes and minor performance issues? We could always bring in additional work in a second PR. (I have more features to build off of this foundation myself, but not in this PR.)